test/unit/lazyAndCommands/lazy-devices-test.js

```diff
@@ -169,7 +169,9 @@ describe('IoT Agent Lazy Devices', function() {
             async.series([
                 apply(iotAgentLib.activate, iotAgentConfig),
                 apply(iotAgentLib.register, device1)
-            ], done);
+            ], function(error) {
```

Comment:
Although it is a test, maybe you would like to leave it as it was before, I mean passing `done` as the callback to `async.series` :)